packages/flutter-webrtc/.gitignore

```diff
@@ -0,0 +1,56 @@
+*.iml
```

Comment:
The package name is `flutter_webrtc` (underscore between the words) but not `flutter-webrtc`. Rename the directory.